lib/sgov/queries/get-class.sparql

```diff
@@ -0,0 +1,16 @@
+PREFIX z: <https://slovník.gov.cz/základní/pojem/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+
+CONSTRUCT {
+  %NODE% a z:typ-objektu ;
```

Comment:
It may be nice to describe the format of the string, i.e. what substitutions are expected to happen as a part of the ```*.sparql``` file loader. It may even be interesting to load those files not as strings but wrap them with some substitution function so the user only provide map of arguments.

Reply:
#51